### ISSUE TITLE
Navbar

### ISSUE THREAD

**Issue by vorcigernix** [Developer-DAO/ddrpc-frontend#2]
This is Dashboard  navbar. Should contain D_D logo, My Projects and My Account links (correct @kempsterrrr ?)

**Comment by PSkinnerTech** [comment]
There should be 2 separate navbars.
- 1 for the dashboard
- 1 for the home page

The homepage navbar, I would recommend be a component sourced from the `/components` folder in the root of the project.

The dashboard navbar is currently in the `/app/_components` folder.

**Comment by vorcigernix** [comment]
Yup. Still haven't figured out the issue with github projects, so I can't turn this into subtask. I'll try to describe it at least in the issues.

**Comment by PSkinnerTech** [comment]
Do you see this?

https://github.com/orgs/Developer-DAO/projects/14/views/2

**Comment by vorcigernix** [comment]
Yup
